query03.sql

```diff
@@ -0,0 +1,18 @@
+/* query 03
+Using the Philadelphia Water Department Stormwater Billing Parcels dataset, pair each parcel with its closest bus stop. 
+The final result should give the parcel address, bus stop name, and distance apart in meters. Order by distance (largest on top).
+*/
+
+SELECT
+      p.address,
+      c.stop_name,
+      c.distance * 111000 AS distance_m
```

Comment:
When you're measuring distances with geometries you always want to use a non-lat/lng projection.